lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -1434,6 +1452,15 @@ sub merge
                 $medium_merge->{new_id},
                 @{$medium_merge->{old_ids}},
             );
+            $self->c->sql->do(<<~'SQL', $medium_merge->{new_id}, [@{$medium_merge->{old_ids}}]);
```

Comment:
this is dereferencing `$medium_merge->{old_ids}` and then turning it back into an array ref (making a shallow copy), but I don't think that's needed for passing to `do`, which doesn't modify its parameters
```suggestion
            $self->c->sql->do(<<~'SQL', $medium_merge->{new_id}, $medium_merge->{old_ids});
```